### PR TITLE
ADD PR template

### PR DESCRIPTION
**Pull request by yashdev9274**
I have added PR template that looks like this

image

### CODE REVIEW

**Comment by yanukadeneth99** [comment]
Hey @yashdev9274, thanks for the addition! I think this is great. I'll go through this properly and get back to you 😄

File: .github/PULL_REQUEST.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
+## Fixes Issue
```
Comment (yanukadeneth99) [inline .github/PULL_REQUEST.md:3]: We can remove this section since we don't rely much in GitHub Issues.

File: .github/PULL_REQUEST.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+<!-- Example: Closes #31 -->
+
+## Changes proposed
+
+<!-- List all the proposed changes in your PR -->
+
+<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->
+<!--
+[x] - Correct; marked as done
```
Comment (yanukadeneth99) [inline .github/PULL_REQUEST.md:15]: Maybe we can add few examples here.

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]
I think this looks great, however, @haardikk21 what do you think of something like this since y'all are gonna move away from GitHub eventually?

**Comment by yashdev9274** [comment]
> I think this looks great, however, @haardikk21 what do you think of something like this since y'all are gonna move away from GitHub eventually?

Move away from GitHub??? What does that mean???

**Comment by yashdev9274** [comment]
Hey @yanukadeneth99 any update on this PR?

**Comment by yanukadeneth99** [comment]
Hey @yashdev9274, I'm waiting for a word from @haardikk21. He's extremely busy with the new V2 coming out. 😉

**Comment by haardikk21** [comment]
This is a good change for the meantime, but yes starting with v2 we will be shifting off Github.

**Review by yanukadeneth99 — CHANGES_REQUESTED** [review]
Could you do the changes suggested?

**Comment by yashdev9274** [comment]
> Could you do the changes suggested? @yashdev9274

yes I'll make the required changes

**Comment by yanukadeneth99** [comment]
Any updates @yashdev9274?

**Comment by yanukadeneth99** [comment]
Closing this for inactivity. Tag me to re-open this back.

**Pull request closed without merging**